src/discussions/learners/data/selectors.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable import/prefer-default-export */
+
+import { createSelector } from '@reduxjs/toolkit';
+
+export const selectAllLearners = createSelector(
+  state => state.learners,
+  learners => learners.learners,
+);
+
+export const learnersLoadingStatus = () => state => state.learners.status;
+
+export const selectLearnerSorting = () => state => state.learners.sortedBy;
+
+export const selectLearnerFilters = () => state => state.learners.filters;
+
+export const selectLearnerNextPage = () => state => state.learners.nextPage;
+
+export const selectLearnerAvatar = author => state => (
+  state.learners.learnersProfile.find(l => l.username === author)?.profile_image.image_url_small
+);
+
+export const selectLearnerLastLogin = author => state => (
+  state.learners.learnersProfile.find(l => l.username === author)?.last_login
+);
```

Comment:
Store these by username so this can be as simple as `learnerProfiles[username]`

Reply:
Done